src/engraving/libmscore/navigate.cpp

```diff
@@ -372,13 +372,13 @@ ChordRest* Score::upStaff(ChordRest* cr)
 ChordRest* Score::downStaff(ChordRest* cr)
 {
     Segment* segment = cr->segment();
-    int tracks = nstaves() * VOICES;
+    TrackIdx tracks = nstaves() * VOICES;
 
-    if (cr->staffIdx() == static_cast<int>(nstaves()) - 1) {
+    if (cr->staffIdx() == nstaves() - 1) {
```

Comment:
Will this code work correctly if the staff list is empty?

Reply:
it will be the same as it was before